content/en-us/luau/control-structures.md

```diff
@@ -166,6 +166,32 @@ The generic `for` loop iterates over items in a collection rather than a sequenc
 
 For loops need a function, or iterator, to iterate over different types of collections. The global `ipairs()` returns an iterator for arrays, and the global `pairs()` returns an iterator for dictionaries. The `Library.string` library provides `Library.string.gmatch()` to iterate over strings.
 
+### Generalized Iteration
+
+In Luau, it is possible to iterate over a table by using the `in` keyword directly on the table, instead of using an iterator function such as `ipairs()`.
```

Comment:
```suggestion
In Luau, you can iterate over a table using the `in` keyword directly on the table, instead of using an iterator function such as `ipairs()`:
```